samples/cpp/CMakeLists.txt

```diff
@@ -235,7 +235,7 @@ macro(ie_add_sample)
 
     find_package(OpenVINO REQUIRED COMPONENTS Runtime)
     if(c_sample)
-        set(ov_link_libraries openvino::runtime::c)
+        set(ov_link_libraries openvino::runtime::ov openvino::runtime::c)
```

Comment:
Why do we need to have `openvino::runtime::ov`

Reply:
The name is very strange `openvino::runtime::ov` is reading like `openvino runtime openvino`.
From my point of view we need to remove or change library name `openvino::runtime::ov`

moreover c samples should depend only from `openvino::runtime::c`